tests/test_config.py

```diff
@@ -76,9 +77,11 @@ def test_config_load_from_file(self):
     def setUp(self):
         # clear `config_load`-relevant environment variables before testing, so
         # we only need to patch the ones that we are currently testing
```

Comment:
I could've swore this said `empty=True` when I first looked and just when I was about to comment on it it switched. Now I don't know what I saw anymore. 😅

Reply:
I said empty at first, but that was a mistake on my part. Forgot later tests actually depend on env vars like `HOME` on linux or `SYSTEMDRIVE` on windows...